packages/plugin-core/src/constants.ts

```diff
@@ -902,6 +902,12 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     },
     when: "dendron:pluginActive",
   },
+  LOCK_PREVIEW: {
+    key: "dendron.lockPreview",
+    title: `${CMD_PREFIX} Lock Preview`,
+    icon: `$(open-preview)`,
```

Comment:
can we get a different icon? see https://code.visualstudio.com/api/references/icons-in-labels

Reply:
[`75aa659` (#3293)](https://github.com/dendronhq/dendron/pull/3293/commits/75aa6598d30ca0be5dc4efdb3ebef196d764727a)